config/manager/manager.yaml

```diff
@@ -40,9 +40,6 @@ spec:
         - /manager
         args:
         - "--webhook-server-port=9443"
-        # If /config/default/manager_auth_proxy_patch.yaml is used, the port in
-        # --metrics-addr must match the value of --upstream passed to
-        # kube-rbac-proxy.
         - "--metrics-addr=:8080"
```

Comment:
Wouldn't just changing this to 8443 fix our issue and still allow us to use HTTPS? I'm not sure we want to take a step back in security and not use HTTPS for metrics.

Reply:
> Wouldn't just changing this to 8443 fix our issue and still allow us to use HTTPS? I'm not sure we want to take a step back in security and not use HTTPS for metrics.

No, there is nothing providing a HTTPS listener (on port 8443). Then we need to include the kube-rbac-proxy sidecar and route all metrics scrape request through its HTTPS listener - forwarding the request to the controller-manager metrics HTTP endpoint. That is an alternative mentioned in the PR description, though more complex, and I don't see the true benefit of this. I expect many K8s distros to block namespace ingress traffic by default.

Reply:
Yeah we took out kube-rbac-proxy a while ago (it has a bunch of problems from a supply chain perspective) and I'd rather not put it back in without good justification.

Reply:
We don't need rbac proxy for https, that is only needed for authz. We already serve https traffic for the webhook on 9443. So I guess I don't see why we couldn't also use it (or have the option) for 8443 for metrics. However, that is probably out of scope of this PR if metrics are currently broken. 

I created [this](https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/204) to revisit this.